core/src/main/java/io/grpc/ChannelImpl.java

```diff
@@ -376,6 +397,10 @@ public void transportTerminated() {
   public static final Metadata.Key<Long> TIMEOUT_KEY =
       Metadata.Key.of(HttpUtil.TIMEOUT, new TimeoutMarshaller());
 
+  // TODO(carl-mastrangelo): move this to internal
+  public static final Metadata.Key<String> MESSAGE_ENCODING_KEY =
+      Metadata.Key.of(HttpUtil.MESSAGE_ENCODING, Metadata.ASCII_STRING_MARSHALLER);
```

Comment:
Should we add this to HttpUtil (under internal)?  We should probably rename it to `GrpcUtil` since it's basically set of all purpose constants.  I've actually done this in #680.